certora/specs/summaries-Folio.spec

```
@@ -0,0 +1,4 @@
+// No summaries needed for Folio - no matching function calls detected
```

RuboCop (1.87.0)

[fatal] unexpected token tCONSTANT
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)